reloj-checador/spec/helpers/attendaces_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the AttendacesHelper. For example:
+#
+# describe AttendacesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
delete the comments if they are not necessary anymore

Reply:
[https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08/pull/16/commits/cd57fa3b0805cd022c7fa33f759d0eb8ebed9474](removed the comments)